CIP-XXXX/CIP-XXXX.md

```diff
@@ -0,0 +1,916 @@
+focussed web-based stacks. Here we aim to support the requirements of both Ada
+holders' and DReps' interactions with such web-based stacks.
+
+> **Note** This proposal assumes knowledge of the ledger governance model as outlined within
+> [CIP-1694?](https://github.com/cardano-foundation/CIPs/pull/380).
+
+## Motivation: why is this CIP necessary?
+
+CIP-1694 introduces many new concepts, entities and actors to Cardano;
+describing their implementation at the ledger level. Yet, for most eco-system
+participants low level details are abstracted away by tools, such as wallets.
+This creates a need for tooling to be able support the utilization of ledger
+features. This specification allows for creation of web-based tools for the
+utilization of CIP-1694's governance features.
+
+This proposal enables Ada holders and DReps to engage web-based tooling through
+wallets. Thus the primary stakeholders for this proposal are tool developers and
+wallet providers. Here we aim to outline all endpoints needed to be exposed to
+web based tools to support all the needs Ada holders and DReps to engage with
+CIP-1694's governance design.
+
+## Specification
+
+We define the following specification as an extension to the specification
+described within CIP-30.
+
+> **Note** This specification will evolve as the proposed ledger governance
+> model matures. It is likely the precise data structures outlined here will be
+> need to be adjusted.
+
+### DRep Key
+
+CIP-1694 does not define a derivation path for registered DRep credentials, here
+we propose the introduction of DRep Keys to act as DRep credentials for
+registered DReps.
+
+The methods described here should not be considered the only definitive method
+of generating DRep credentials. Rather these methods should be employed to
+derive keys for non-script registered DReps.
+
+#### Derivation
+
+Here we describe DRep Key derivation as it attains to Cardano wallets who follow
+the
+[CIP-1852 | HD (Hierarchy for Deterministic) Wallets for Cardano](https://github.com/cardano-foundation/CIPs/blob/master/CIP-1852/README.md)
+standard.
+
+To differentiate DRep credentials from other Cardano keys the derivation path
+must follow:
+
+`m / 1718' / 1815' / account' / chain / address_index`
+
+We strongly suggest that a maximum of one set of DRep credentials should be
+associated with one wallet account, this can be achieved by setting `chain=0`
+and `address_index=0`. Thus avoiding the need for DRep Key discovery.
+
+We believe the overhead that would be introduced by "multi-DRep" accounts is an
+unjustified expense. Future iterations of this specification may expand on this,
+but at present this is seen as unnecessary.
+
+> **Note** `1718` was the year that François-Marie adopted the pseudonym Voltaire.
+
+#### Tooling
+
+Supporting tooling should clearly label these key pairs as "CIP-???? DRep Keys".
+
+Bech32 prefixes of `drep_sk` and `drep_vk` should be used.
```

Comment:
I don't see these in CIP-0005... should a CIP-0005 modification to add these prefixes therefore be included in this PR?

Reply:
Good spot @rphair, I have added placeholders in.

Reply:
resolved in https://github.com/cardano-foundation/CIPs/pull/509/commits/65e34ef65214752adfaf9ac31d67b77297191184